test/test_helper.rb

```diff
@@ -22,13 +22,10 @@ class ActiveSupport::TestCase
   include FactoryBot::Syntax::Methods
 
   before do
-    Bullet.start_request
```

Comment:
This shuts off bullet in non-http requests -- stuff that doesn't spoof a rails server. This was throwing all sorts of obnoxious false positives.